ouroboros-consensus/docs/interface-CHANGELOG.md

```diff
@@ -56,12 +56,45 @@ may appear out of chronological order.
 The internals of each entry are organized similar to
 https://keepachangelog.com/en/1.1.0/, adapted to our plan explained above.
 
+## Circa 2022-04-26
+
+### Added
+
+- New 'Praos' protocol implemented in `Ouroboros.Consensus.Protocol.Praos`. This
+  uses only a single VRF proof in the header, and drops support for the overlay
+  schedule (so drops support for changing the decentralisation parameter). All
+  of the relevant keys etc remain the same.
+- A new era `BabbageEra` has been created, and included in the structure of
+  `CardanoEras`. Anything using `CardanoEras` will now need an additional case
+  for Babbage.
```

Comment:
I think a pointer to/mention of the Babbage pattern synonyms would be useful here.

Reply:
Clarke added it 👍